Sources/xcproj/XCConfig.swift

```diff
@@ -93,8 +93,14 @@ final class XCConfigParser {
                                                             length: line.count))
             .compactMap { (match) -> (key: String, value: String)?  in
                 if match.numberOfRanges == 3 {
-                    let key: String = NSString(string: line).substring(with: match.range(at: 1))
-                    let value: String = NSString(string: line).substring(with: match.range(at: 2))
+                    let key: String = (line as NSString).substring(with: match.range(at: 1))
+                    var value: String = (line as NSString).substring(with: match.range(at: 2))
+
+                    // strip the trailing semicolon, if any
+                    if value.hasSuffix(";") {
```

Comment:
What about using this regular expression? `.+(;\s*)$`? It should also catch the cases where there are spaces right after the semicolon.

Reply:
Any whitespace before the semicolon could also be stripped right?

Reply:
We can then use `.+(\s*;\s*)$`

Reply:
So the problem was if the semi-colon was optional then the `.*` would consume everything. So I updated that to `.*?` (dunno why I didn't think of that yesterday) and now it works as expected. Thanks!